paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/SupportedPaymentMethod.kt

```diff
@@ -414,6 +414,17 @@ internal sealed class SupportedPaymentMethod(
             // AuBecsDebit // TODO: uncomment once we are ready to go live
         )
 
+        /**
+         * This is a list of payment methods that we should tint their icons
+         * when they are selected in UI.
+         */
+        internal fun SupportedPaymentMethod.shouldTintOnSelection(): Boolean {
+            return listOf(
+                Card,
+                AuBecsDebit
```

Comment:
Can we get a screenshot of AuBecsDebit? Where can we find information about the payment methods that need to be tinted on selection?